src/Job/NormalJob.ts

```diff
@@ -79,7 +79,26 @@ export class NormalJob implements Job {
 	}
 
 	/**
-	 * Returns the number of prerequisite Jobs using Set's native size property.
+	 * Gets this Job's dependencies (its prerequisites, recursively)
+	 *
+	 * @returns a iterable containing a deep scan of the Job's prerequisites.
+	 */
+	public getDeepPrerequisitesIterable(): Array<Job> {
+		let childrenPrereqs: Array<Job> = []
```

Comment:
I find the name `childrenPrereqs` kinda misleading since it holds more than just the children prerequisites by the end of the function.